screens/resources/ResourceScreen.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { TopText } from '../../styled/resources';
+import { TouchableOpacity, ScrollView, View, Text, Button } from 'react-native';
+import getResources from '../../lib/resourceUtils';
+import ResourceCard from '../../components/resources/ResourceCard';
+import ResourceTab from '../../components/resources/ResourceTab';
+
+class ResourceScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      resources: [],
+      DCCentralKitchenResources: [],
+      CommunityResources: [],
+      GovernmentResources: [],
+      ResourcesForSeniors: []
+    };
+  }
+
+  async componentDidMount() {
+    getResources().then(resources => {
+      (DCCentralKitchenResources = resources.filter(
```

Comment:
Is there any reason why these are all wrapped in parentheses? I think it could be easily written as `const dcCentralKitchenResources = resources.filter()`... etc on separate lines (not comma-separated). But maybe there's a specific reason parentheses used it here?